### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by calispac**
changing some links to the organization repo

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,7 +1,7 @@
-https://github.com/dneise/protozfitsreader/archive/v0.43.1.tar.gz
+https://github.com/cta-sst-1m/protozfitsreader/archive/v0.43.1.tar.gz
 -e git+https://github.com/cta-observatory/pyhessio.git#egg=pyhessio
 -e git+https://github.com/cta-observatory/ctapipe.git#egg=ctapipe
 -e git+https://github.com/cta-observatory/ctapipe-extra.git#egg=ctapipe-extra
 -e git+https://github.com/yrenier/CTS.git#egg=CTS
```
Comment (dneise) [inline requirements.txt]: Are we also going to move CTS to the organisation repo soon? In that case let us wait for CTS to move and then also modify this line accordingly

Comment (calispac) [reply requirements.txt]: Ok fine ! @yrenier  do you want to move CTS to cta-sst-1m repo?

Comment (yrenier) [reply requirements.txt]: Sure I'm looking how to do it

Comment (yrenier) [reply requirements.txt]: done

Comment (dneise) [reply requirements.txt]: That was quick :+1:

**Pull request merged**